index.html

```diff
@@ -428,6 +499,10 @@ <h2>
           each Gamepad present at the index in the array specified by its
           {{Gamepad/index}} attribute. Array indices for which there is no
           connected Gamepad with the corresponding index should return null.
+
+          <p>
+          A new array MUST be returned for each call to getGamepads().
```

Comment:
We can delete this and annotate getGamepads() with [`[NewObject]`](https://heycam.github.io/webidl/#NewObject).